command/status.go

```diff
@@ -23,7 +26,13 @@ Usage: nomad status [options] <identifier>
 
 General Options:
 
-  ` + generalOptionsUsage()
+  ` + generalOptionsUsage() + `
+
+Status Options:
+
+  -verbose
+    Display full information.
```

Comment:
should `-verbose` be added to `AutoCompleteFlags`? for example, like in `job run`: 
https://github.com/hashicorp/nomad/blob/79132127b92787e3a7513ba9382388eaba7a82e7/command/job_run.go#L106

Reply:
Good call 👍